burn_boise.rb

```diff
@@ -11,31 +12,125 @@ def get_aqi(zipcode)
   aqi
 end
 
+class Regulations
```

Comment:
Might be better to move this to `lib/regulations.rb` and `require` it at the top

Reply:
It sneakily uses `get_aqi`